docker_image_build_files/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# a Node.js base image that includes Node v18
+FROM node:18-bullseye
+
+# Install Git and Java JDK and jq
+RUN apt-get update && \
+    apt-get install -y git default-jdk jq && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# RUN nvm install 18.19.0
+# RUN nvm use 18.19.0
+RUN mkdir -p /home/user/projects/
+WORKDIR /home/user/projects/
+
+
+# # Clone GitHub repository
```

Comment:
Is this normal to clone and install git?  I haven't noticed it elsewhere.

Reply:
The Ideal would be to use Attachment Volumes, but that would make it more complicated i feel
Cloning to git works especially if just adopt working with remote containers , that way we encourage committing early and often